pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -103,13 +103,6 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 					updateFail(r, ctx, obj, &err, r.Client, nil)
 					return common.ResultRequeueAfter10sec, err
 				}
-				vpcNetworkConfiguration := &v1alpha1.VPCNetworkConfiguration{}
```

Comment:
Is this a rebase conflict issue we keep these code here?

Reply:
I am not sure why add these lines here, it seems introduced in #705 (https://github.com/vmware-tanzu/nsx-operator/pull/705/files#r1734297594), it’s already got a vpcNetworkConfiguration in line 83, I think it’s redundant here, so deleted it.

```
		vpcNetworkConfiguration := &v1alpha1.VPCNetworkConfiguration{}
		err = r.Client.Get(ctx, types.NamespacedName{Name: commonservice.SystemVPCNetworkConfigurationName}, vpcNetworkConfiguration)
		if err != nil {
			log.Error(err, "failed to get system VPCNetworkConfiguration")
			updateFail(r, ctx, obj, &err, r.Client, nil)
			return common.ResultRequeueAfter10sec, err
		}
```